packages/react/src/components/Footer/Footer.js

```diff
@@ -16,6 +17,7 @@ import LegalNav from './LegalNav';
 import LocaleButton from './LocaleButton';
 
 const { prefix } = settings;
+const cPrefix = carbonSettings.prefix;
```

Comment:
My thoughts from Slack. Do with it what you will.

> @kennylam I would almost rather keep `prefix = bx` since that’s what Carbon uses, and I would rather the stable selector’s prefix be something different like `stablePrefix = dds`. Maybe the same with settings `{ settings as ddsSettings }`.

Reply:
@photodow I missed this. PR was practically up already 😭

Reply:
I figured it was the case. No worries, sorry I commented so late.

Reply:
Np. @jeffchew and @annawen1 both agree with your suggestion. I do too, so i'll get it in.